### DISCUSSION TITLE
"binding.node" and "fse.node" can't be opened?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#529] [Get Help] [answered]
Help! Not sure what this means! They pop up after I run quire preview and I'm able to see my Quire preview, but the TOC sidebar doesn't work, some of the buttons are missing, and the plate images are missing. Can someone advise please?

![Screen Shot 2022-10-06 at 2 27 02 PM](https://user-images.githubusercontent.com/74433007/194402147-b4e7758f-5da5-485d-aa9a-99c4486cb402.png)

![Screen Shot 2022-10-06 at 2 27 51 PM](https://user-images.githubusercontent.com/74433007/194402161-fbc02581-326c-44dc-a521-8cbd4ed6d0b0.png)

**Comment by mmpadilla15** [comment, marked as answer]
OK ... I'm not entirely sure what happened, but I restarted my computer then went into the security settings to "allow" binding.node and fse.node and then ran `quire preview` again. it all seems to be working now. Fingers crossed!

**Reply by Erin-Cecele** [reply]
@mmpadilla15 Hi Michelle, seems like you answered your own question. Is this still working ok for you?

**Reply by mmpadilla15** [reply]
@Erin-Cecele Yes - It seems to be!
